core/src/main/java/org/neo4j/graphalgo/core/heavyweight/HeavyGraph.java

```diff
@@ -130,8 +130,8 @@ public int getLabel(int nodeId) {
     }
 
     @Override
-    public int getEdgeLabel(long nodeId1, long nodeId2) {
-        AbstractMap.SimpleEntry<Long, Long> key = new AbstractMap.SimpleEntry<>(nodeId1, nodeId2);
+    public int getEdgeLabel(Integer nodeId1, Integer nodeId2) {
+        AbstractMap.SimpleEntry<Integer, Integer> key = new AbstractMap.SimpleEntry<>(nodeId1, nodeId2);
         Integer label = labelMap.getValue().get(key);
         if (label == null) {
             key = new AbstractMap.SimpleEntry<>(nodeId2, nodeId1);
```

Comment:
Pair?

Reply:
Its in a different package. We don't have acess to Pair here.